content/My favorite extensions for programming Ruby in Visual Studio Code.md

```diff
@@ -0,0 +1,106 @@
+# Choosing a text editor
+
+It may be a tough decision, as a programmer, the text editor that you choose will be your main tool to do your work, and you will use it every day in your work-life.
```

Comment:
`It may be a tough decision, as a programmer, to choose a text editor, since it will be your main tool to do your work, and you will use it every day in your work-life.`

Maybe something like this? Because I feel you were trying to say that is a tough decision to choose a text editor but you actually say something is a tough decision (don't know what) and explain why a text editor is important.